### DISCUSSION TITLE
Book Search

### DISCUSSION THREAD

**Discussion by barrelltitor** [gotson/komga#1616] [Ideas]
Series are great and all, but there's many tie in moments especially for older Marvel & DC Comics, and while Reading Lists are great, they're incomplete and manually making them is such a chore. 

At the moment you can search by character, but just on a Series level. If you could search on a book level and make a Reading List out of it the process would be simplified. The book metadata already exists! 

Combine this with the smart playlists feature idea and you'd have the best comic library management out there

**Comment by gotson** [comment]
i don't understand what the request is about, can you clarify?

**Reply by barrelltitor** [reply]
Searching for comic book. Books(issues), not just series.
